### ISSUE TITLE
Create accessibility checklist for Learn content and meeting notes

### ISSUE THREAD

**Issue by kaitohm** [WordPress/Learn#1663]
The team is looking for volunteers who can work on a new handbook page for the team regarding accessibility. We’d like to create a checklist that can be used by content creators, editors, and meeting note-takers. Ann started drafting ideas [towards an accessibility checklist for junior WordPress developers](https://docs.google.com/document/d/1qsNhlbrJrUR7g0ngk0O-y9m3sfG48MyWXlhYpD9Yizo/edit?usp=sharing).

Ann N, @jomarieminney and @alexstine have volunteered to work on this: https://wordpress.slack.com/archives/C02RW657Q/p1687850702693869

**Comment by kaitohm** [comment]
This handbook page was published here: [Training Team Accessibility Checklist](https://make.wordpress.org/training/handbook/guidelines/training-team-accessibility-checklist/)
